haralyzer/assets.py

```diff
@@ -341,11 +341,11 @@ def filter_entries(self, request_type=None, content_type=None,
         return results
 
     def get_load_time(self, request_type=None, content_type=None,
-                      status_code=None, async=True):
+                      status_code=None, **kwargs):
```

Comment:
So sorry, I should have been more clear about this. What you originally did was perfect (using the keyword argument `asynchronous`, as this is what people should be doing moving forward. However, I was thinking we would use `kwargs` to catch the old `async` keyword, allowing both to be used. This way, we could eventually fully deprecate the old `async` keyword argument. Does that make sense?

Reply:
Haha, yes. I'll give it a 3rd go later.

Reply:
Lol, so sorry. I'm happy to take over from here if it is getting annoying!